roles/kubernetes/control-plane/tasks/kubeadm-etcd.yml

```diff
@@ -12,20 +12,6 @@
   tags:
     - network
 
-- name: Ensure etcdctl binary is installed
-  include_tasks: "{{ role_path }}/../../etcd/tasks/install_host.yml"
-  vars:
-    etcd_cluster_setup: true
-  when: etcd_deployment_type == "host" and not etcd_kubeadm_enabled
-
-- name: Ensure etcdctl binary is installed
-  include_tasks: "{{ role_path }}/../../etcd/tasks/install_etcdctl_docker.yml"
-  vars:
-    etcd_cluster_setup: true
-    etcd_retries: 4
-  when:
-    - etcd_deployment_type == "docker" and not etcd_kubeadm_enabled
-
```

Comment:
looks good for me as https://github.com/kubernetes-sigs/kubespray/blob/5a351b4b00b0ad334c0c31184f42809a5a159930/roles/kubernetes/control-plane/tasks/main.yml#L66-L68